File: CodeWars/8Kyu/2.SentenceSmash/Python/SentenceSmash.py
def main():
    size = int(input("Enter the size of the array : "))
    words = []

    for i in range(size):
        a = input(f"Enter the {i + 1}th element of the array : ")
        words.append(a)

    result = Smash(words)

    print(result)
    
def Smash(words : str) -> str:
    result = ""

    for i in words:
        result = result + " " + i

    return result

if __name__ == '__main__':
    main()

File: CodeWars/8Kyu/2.SentenceSmash/Typescript/SentenceSmash.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var PromptSync = require("prompt-sync");
var prompt = PromptSync();
function main() {
    var size = Number(prompt("Enter the size of the array : "));
    var words = [];
    for (var i = 0; i < size; i++) {
        var a = prompt("Enter the ".concat(i + 1, "th element of the array : "));
        words.push(a);
    }
    var result = Smash(words);
    console.log(result);
}
function Smash(words) {
    var result = "";
    for (var _i = 0, words_1 = words; _i < words_1.length; _i++) {
        var i = words_1[_i];
        result = result + " " + i;
    }
    return result;
}
main();
